Quaver.Shared/Screens/Gameplay/UI/PauseScreen.cs

```diff
@@ -146,9 +175,12 @@ public void Activate()
             ClearTransformations();
 
             Background.Animations.Add(new Animation(AnimationProperty.Alpha, Easing.Linear, 0, 1, 400));
-            Continue.Animations.Add(new Animation(AnimationProperty.X, Easing.OutExpo, -Continue.Width, GetActivePosX(Continue),  400));
-            Retry.Animations.Add(new Animation(AnimationProperty.X, Easing.OutExpo, -Retry.Width, GetActivePosX(Retry), 400));
-            Quit.Animations.Add(new Animation(AnimationProperty.X, Easing.OutExpo, -Quit.Width, GetActivePosX(Quit), 400));
+            Continue.Animations.Add(new Animation(AnimationProperty.X, Easing.OutExpo, -Continue.Width,
```

Comment:
Not sure if your IDE automatically did this, but we might as well make this better while we're here.

Use `Continue.MoveToX()`. It's a shorter way of doing the same thing. (and for the other ones)

Reply:
Nah, my ide didn't do anything to the code for the animations, i think the reason it showed up as changed code is due to formatting the class file, which I accidently did.

Reply:
But yeah, I will change the code out to MoveToX()